src/main/java/pl/termosteam/kinex/dto/JwtActivationRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package pl.termosteam.kinex.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class JwtActivationRequestDto {
+    private String username;
+    private String activation_token;
```

Comment:
Czy jest jakiś powód nazewnictwa z _ ? Zawsze możecie użyć adnotacji JsonProperty żeby uniknąć powielania dziwnego nazewnictwa w kodzie

Reply:
poprawilem i dodalem  @JsonProperty("activation_token"), to byl jedyny powod :)